api/v4/source/roles.yaml

```diff
@@ -100,7 +100,7 @@
 
         ##### Permissions
 
-        `manage_system` permission is required.
+        Must have `sysconsole_write_user_management_permissions` or `manage_system` permission
```

Comment:
There is a slight nuisance here: In order to update the roles of a system admin, one needs to be a system admin themself. Should we also document that?

Reply:
Updated it now. Can you have a final look before merging?